File: android/src/main/java/com/achatsocial/releep_watch_connect/ScanBLEResponse.java
package com.achatsocial.releep_watch_connect;
import com.google.gson.annotations.SerializedName;

public class ScanBLEResponse {
    @SerializedName("DeviceName")
    private String DeviceName = "";
    @SerializedName("MacAddress")
    private String MacAddress = "";

    public ScanBLEResponse(String deviceName,String macAddress){
        this.DeviceName = deviceName;
        this.MacAddress = macAddress;
    }
}

File: ios/Classes/ScanBLEResponse.swift
//
//  ScanBLEResponse.swift
//  releep_watch_connect
//
//  Created by EP&IT on 3/6/2565 BE.
//

import Foundation

class ScanBLEResponse : Codable {
    var DeviceName:String
    var MacAddress:String
    var UUIDString:String

    init(deviceName: String, macAddress: String,uuidString: String) {
        self.DeviceName = deviceName
        self.MacAddress = macAddress
        self.UUIDString = uuidString
    }

}
